src/data_management/timeseries/locations/LocationForm.tsx

```diff
@@ -294,93 +173,32 @@ const LocationFormModel = (props:Props & PropsFromDispatch & RouteComponentProps
-          placeholder={'- Search and select -'}
-          value={selectedAssetType}
-          valueChanged={valueObj => { 
-            // TODO: remove this ts ignore
-            // @ts-ignore
-            setSelectedAssetType (valueObj);
-          }}
-          options={assetTypeOptions}
-          validated
+        <AssetPointSelection
+          value={values.selectedAsset}
+          valueChanged={value => handleValueChange('selectedAsset', value)}
+          triedToSubmit={triedToSubmit}
           onFocus={handleFocus}
           onBlur={handleBlur}
-        /> */}
-        <MapSelectAssetOrPoint
-          title={'Asset location'}
-          name={'selectedAssetObj'}
-          // placeholder={'- Search and select -'}
-          value={values.selectedAssetObj}
-          valueChanged={(value)=>handleValueChange('selectedAssetObj', value)}
-          validated={true}
-          triedToSubmit={triedToSubmit}
         />
-       
-        
-        <div style={{display: "flex"}}>
-          <div style={{width: "58%", marginRight: "40px"}}>
-            <GeometryField
-              title={'Geometry'}
-              name={'selectedAssetObj'}
-              // placeholder={'- Search and select -'}
-              value={values.selectedAssetObj}
-              valueChanged={(value)=>handleValueChange('selectedAssetObj', value)}
-              validated={true}
-              triedToSubmit={triedToSubmit}
-              onFocus={handleFocus}
-              onBlur={handleBlur}
-            />
-          </div>
-          <div>
-            <label
-              className={formStyles.Label}
-            >
-              <span className={formStyles.LabelTitle}>
-                Selected asset
-              </span>
-              <a 
-                href={
-                  values.selectedAssetObj.asset && 
-                  values.selectedAssetObj.asset.value &&  
-                  values.selectedAssetObj.asset.value.entity_name ? 
-                  `/api/v3/${values.selectedAssetObj.asset.value.entity_name}s/${values.selectedAssetObj.asset.value.entity_id}`
-                  : values.selectedAssetObj.asset?
-                  `/api/v3/${currentRecord.object.type}s/${currentRecord.object.id}`
-                  :
-                  '/api/v3/'
-                }
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {values.selectedAssetObj.asset? values.selectedAssetObj.asset.label : "None selected. See all endpoints" }
-              </a>
-            </label>
-          </div>
-        </div>
-       
-        {/* <TextArea
-          title={'Extra metadata (JSON) *'}
+        <TextArea
+          title={'Extra metadata (JSON)'}
           name={'extraMetadata'}
-          placeholder={'Enter valid JSON'}
+          placeholder={'Please enter in valid JSON format'}
           value={values.extraMetadata}
           valueChanged={handleInputChange}
           clearInput={clearInput}
```

Comment:
The JSON field can for now be commented out since there is a bug with it:
https://github.com/nens/lizard-management-client/issues/658#issuecomment-813976038

Reply:
I check your comment in the other ticket. However, there is no mention of why the JSON field is commented out and which bug it is associated with. Could you check again? I can use the JSON field with no problem.

Reply:
The supplier field is indeed not supported by the API yet so I removed it completely.

Reply:
commented out the JSON field